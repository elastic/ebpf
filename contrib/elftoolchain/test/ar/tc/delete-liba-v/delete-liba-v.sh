# $Id$
inittest delete-liba-v tc/delete-liba-v
extshar ${TESTDIR}
extshar ${RLTDIR}
runcmd "${AR} dv liba.a a1.o a3.o" work true
runcmd "plugin/teraser -ce -t delete-liba-v liba.a" work false
runcmd "plugin/teraser -e liba.a" result false
rundiff true
