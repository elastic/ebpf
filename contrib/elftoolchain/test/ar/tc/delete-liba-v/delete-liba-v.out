d - a1.o
d - a3.o
